resources/styles/components/concept-coach/launcher.less

```diff
@@ -0,0 +1,520 @@
+  }
+  .launcher-text {
+    polygon {
+      &:nth-child(odd) {
+        fill: @launcher-text-dark;
+      }
+      &:nth-child(even) {
+        fill: @launcher-text-dark;
+      }
+    }
+    rect {
+      fill: @launcher-text-dark;
+    }
+  }
+}
+
+.launcher-select-keyframes(@default-rule, @select-rule){
+  0% {
+    @default-rule();
+  }
+
+  5% {
+    @select-rule();
+  }
+
+  20% {
+    @select-rule();
+  }
+
+  25% {
+    @default-rule();
+  }
+
+  100% {
+    @default-rule();
+  }
+}
+
+.keyframes(launcher-circle-select; {
+  @default-circle-select: {
+    fill: @openstax-white;
+  };
+  @circle-select: {
+    fill: @openstax-info;
+  };
+  .launcher-select-keyframes(@default-circle-select, @circle-select);
+});
+
+.keyframes(launcher-letter-select; {
+  @default-letter-select: {
+    fill: @launcher-circle-text;
+  };
+  @letter-select: {
+    fill: @openstax-white;
+  };
+  .launcher-select-keyframes(@default-letter-select, @letter-select);
+});
+
+.keyframes(launcher-letter-dark-select; {
+  @default-letter-select: {
+    fill: @launcher-letter-dark;
+  };
+  @letter-select: {
+    fill: @openstax-white;
+  };
+  .launcher-select-keyframes(@default-letter-select, @letter-select);
+});
+
+.keyframes(launcher-text-odd-select; {
+  @default-text-select: {
+    fill: @launcher-text-dark;
+  };
+  @text-select: {
+    fill: @launcher-text-selected-dark;
+  };
+  .launcher-select-keyframes(@default-text-select, @text-select);
+});
+
+.keyframes(launcher-text-even-select; {
+  @default-text-select: {
+    fill: @launcher-text-light;
+  };
+  @text-select: {
+    fill: @launcher-text-selected-light;
+  };
+  .launcher-select-keyframes(@default-text-select, @text-select);
+});
+
+.launcher-animate-concept(@name, @delay, @duration, @timing-function: ease-in-out){
+  .animation(~'@{name} @{duration} @{timing-function}');
+  .animation-name(@name);
+  .animation-duration(@duration);
+  .animation-delay(@delay);
+  .animation-timing-function(@timing-function);
+  .animation-iteration-count(infinite);
+}
+
+.launcher-concept(selected, @delay: 0s, @duration: 8s) {
+  .launcher-circle {
+    circle {
+      .launcher-animate-concept(launcher-circle-select, @delay, @duration);
+    }
+    .launcher-letter {
+      .launcher-animate-concept(launcher-letter-select, @delay, @duration);
+
+      &-hole {
+        .launcher-animate-concept(launcher-circle-select, @delay, @duration);
+      }
+
+      &-dark {
+        .launcher-animate-concept(launcher-letter-dark-select, @delay, @duration);
+      }
+    }
+  }
+  .launcher-text {
+    polygon {
+      &:nth-child(odd) {
+        .launcher-animate-concept(launcher-text-odd-select, @delay, @duration);
+      }
+      &:nth-child(even) {
+        .launcher-animate-concept(launcher-text-even-select, @delay, @duration);
+      }
+    }
+    rect {
+      .launcher-animate-concept(launcher-text-even-select, @delay, @duration);
+    }
+  }
+}
+
+.keyframes(launcher-answer-enter; {
+  0% {
+    opacity: 0;
```

Comment:
Can you line up the keyframes so scanning is easier (esp since they only have 1-2 rules/mixins in them)?

```css
0% { @default-rule(); }
5% { @select-rule(); }
20% { @select-rule(); }
25% { @default-rule(); }
100% { @default-rule(); }
```

or 

```css
  0% { @default-rule(); }
  5% { @select-rule(); }
 20% { @select-rule(); }
 25% { @default-rule(); }
100% { @default-rule(); }
```